### ISSUE TITLE
add  /api/export/feed route

### ISSUE THREAD

**Issue by touv** [Inist-CNRS/lodex#403]
to view  resources in ATOM format, see http://atomenabled.org/developers/syndication/

resources should be ordered by update time, to view last resource on the first page

**Comment by touv** [comment]
exporter.extension = 'atom';
exporter.mimeType = 'application/atom+xml';
exporter.type = 'file';
exporter.label = 'Atom Feed';

**Comment by touv** [comment]
https://fr.wikipedia.org/wiki/Atom

**Comment by touv** [comment]
```js
function exporter(config, fields, resources, requestedFields, page, perPage, total) {
    const fieldsByName = fields.reduce((acc, field) => ({
        ...acc,
        [field.name]: field,
    }), {});
    const displayedFields = requestedFields.length ? requestedFields : fields.map(({ name }) => name);
    const resourcesHtml = getResourcesHtml(fieldsByName, displayedFields, resources);

    return html`
    <?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">

 <title>Fil d'exemple</title>
 <subtitle>Un titre secondaire.</subtitle>
 <link href="http://example.org/"/>
 <updated>2010-05-13T18:30:02Z</updated>
 <author>
   <name>Paul Martin</name>
   <email>paulmartin@example.com</email>
 </author>
 <id>urn:uuid:60a76c80-d399-11d9-b91C-0003939e0af6</id>

 <entry>
   <title>Des robots propulsés par Atom deviennent fous</title>
   <link href="http://example.org/2003/12/13/atom03"/>
   <id>urn:uuid:1225c695-cfb8-4ebb-aaaa-80da344efa6a</id>
   <updated>2010-04-01T18:30:02Z</updated>
   <summary>Poisson d'avril !</summary>
 </entry>

</feed>
    `;
}
```

**Comment by touv** [comment]
At least export data in the last version.

**Comment by parmentf** [comment]
The aim of this feed is to syndicate/display on another site the data (in the last version), *not* to display all changes.

`title` should contain the `title` for the Overview tab, and `summary`, its `description`.

Export only the few (`perPage` from `config.json`) first (most recent) resources.

**Comment by parmentf** [comment]
Maybe use OpenSearch to output `totalResults` (as the number of resources in the dataset).

Example from OpenGrey: http://opengrey.eu/search/xml?q=
```xml
<feed xmlns="http://www.w3.org/2005/Atom" xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/">
 <id>23359598185905420</id>
 <updated>2017-07-28T15:12:12+02:00</updated>
 <opensearch:startIndex>0</opensearch:startIndex>
 <opensearch:itemsPerPage>10</opensearch:itemsPerPage>
 <opensearch:totalResults>2000</opensearch:totalResults>
 <title type="text">Results of the OpenGrey search: </title>
 <link type="text/xml" href="http://opengrey.eu/search/request?q="/>
 <opensearch:Query opensearch:searchTerms="" opensearch:startPage="0" opensearch:role="request"/>
 <entry>
  <id>source_og#/content#/681</id>
  <published>2003-06-01T00:00:00+02:00</published>
  <updated>2003-06-01T00:00:00+02:00</updated>
  <title type="text">A positioning system : Galileo A technical scientific strategic ...</title>
  <link type="text/html" href="http://opengrey.eu/handle/10068/681"/>
 </entry>
```

**Comment by QuanticPotatoes** [comment]
fixed in #423
